### DISCUSSION TITLE
Export / import filtered index results to csv or excel

### DISCUSSION THREAD

**Discussion by futurewebsites** [area17/twill#1176] [General]
Are there any import export features? lets say we have an index view of products which can be filtered by brand and category is it possible to export selected columns for the filtered results to a csv file edit the csv file and re-import updating just those selected columns?

**Comment by pboivin** [comment]
Hey @futurewebsites, I'm not aware of any Twill-specific import/export packages. The selective import you're describing sounds like a bit of custom work but for export, I'm thinking you could rely on a Laravel package, such as:

- https://github.com/Maatwebsite/Laravel-Excel
- https://github.com/rap2hpoutre/fast-excel

Both packages seem to offer import, to some extent. Hope this can help!

**Reply by futurewebsites** [reply]
I see there is a bulkExportData function but not sure what it does?

**Reply by ifox** [reply]
It does... nothing currently. It hasn't yet been implemented.
